pkgs/applications/misc/sway-contrib/default.nix

```diff
@@ -15,12 +15,12 @@
 }:
 
 let
-  version = "unstable-2023-06-30";
+  version = "unstable-2024-01-20";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

correct version should be `0-unstable-2024-01-20`

Reply:
no worries, can you also update PR title ?